rosa_architecture/rosa_policy_service_definition/rosa-instance-types.adoc

```diff
@@ -1,29 +1,17 @@
 
-// Module included in the following assemblies:
-//
-// * rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc
-// * rosa_architecture/rosa_policy_service_definition/rosa-hcp-service-definition.adoc
-
-ifeval::["{context}" == "rosa-hcp-service-definition"]
-:rosa-with-hcp:
-endif::[]
-
-:_mod-docs-content-type: CONCEPT
-[id="rosa-sdpolicy-instance-types_{context}"]
-= Instance types
-
-ifdef::rosa-with-hcp[]
-All {hcp-title} clusters require a minimum of 2 worker nodes. All {hcp-title} clusters support a maximum of 90 worker nodes. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss.
-endif::rosa-with-hcp[]
-ifndef::rosa-with-hcp[]
+:_mod-docs-content-type: ASSEMBLY
```

Comment:
Can you re-add the metadata to the top of these module? You can run the script in `./scripts` to get it.

Reply:
should be there now